src/dexter/logger.cr

```diff
@@ -51,7 +51,7 @@ class Dexter::Logger < Logger
 
   # :nodoc:
   def formatter=(value) : Void
-    {% raise "Use log_formatter= instead" %}
+    p "Use log_formatter= instead"
```

Comment:
This can be done at compile time like this: `{% p "Use log_formatter= instead" %}`.

Reply:
Done